examples/kubernetes/servicemesh/mutual-auth-example.yaml

```diff
@@ -31,10 +31,10 @@ spec:
         app: echo
     spec:
       containers:
-      - image: gcr.io/kubernetes-e2e-test-images/echoserver:2.2
+      - image: gcr.io/k8s-staging-ingressconformance/echoserver:v20221109-7ee2f3e
```

Comment:
OK, I see that this echoserver is being used in K8s' gateway-api ([here](https://github.com/kubernetes-sigs/gateway-api/blob/bfbf87882b6bb83e7bf0846aca29d783002690a5/conformance/base/manifests.yaml#L97)), and the seems to be [located here](https://github.com/kubernetes-sigs/ingress-controller-conformance/tree/master/images/echoserver) ([this discussion](https://github.com/kubernetes-sigs/gateway-api/issues/1295) seems also relevant, for the record).